rpcs3/rpcs3qt/tooltips.h

```diff
@@ -25,12 +25,12 @@ class Tooltips : public QObject
 		const QString lib_default_lle              = tr("Select to HLE. (LLE by default)");
 
 		const QString debug_console_mode           = tr("Increases the amount of usable system memory to match a DECR console and more.\nCauses some software to behave differently than on retail hardware.");
-		const QString silence_all_logs             = tr("Stop writing any logs after game startup. Don't use unless you believe it's necessary.");
+		const QString silence_all_logs             = tr("Stop writing any logs after game startup./nRequired for LittleBigPlanet titles due to peformance issues./nDon't use unless you believe it's necessary.");
```

Comment:
Again, including text in a setting description for one game needing a specific setting.. why? We have a wiki for this explaining what game settings to use. If you write about LittleBigPlanet needing the setting but not all the other games that need it, it even implies that no other games need it.